colour-changing-grid/index.html

```diff
@@ -1,14 +1,73 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Colour changing grid</title>
-<link rel="stylesheet" href="style.css">
-<script src="script.js"></script>
+	<title>Colour changing grid</title>
+	<link rel="stylesheet" href="style.css">
 </head>
-
 <body>
-
-Colour changigng grid HTML here
-
+	<!-- Colour changigng grid HTML here -->
+	<div id="container">
+		<div class="sq" onmouseover="mOver(this)" onmouseout="mOut(this)"></div>
```

Comment:
So there are some good pros and cons to doing this repetitive markup here.

+ Performance: it's much faster than JS to have your markup in place and ready to render before JS loads

- Maintenance: What if we need to add a few hundred more for some reason? What if we need to change the classes on every other element, or something similar?

It's important to understand that so you and the teams you work for can make a good decision. It's possible this is the right way for this project, but you never know.

Side note, you can iterate over all these elements in JS and add the listeners so you don't have to maintain it in markup.